### PR TITLE
Hec 898 expired page

### PR DESCRIPTION
**Pull request by priyankadhiman3006**
ticket -> https://jira.tools.tax.service.gov.uk/browse/HEC-898

### CODE REVIEW

**Comment by priyankadhiman3006** [comment]
I have included some test cases missed earlier

File: test/uk/gov/hmrc/heclicensingbodyfrontend/controllers/TaxCheckResultControllerSpec.scala [hunk 1]
```diff
@@ -67,24 +67,17 @@ class TaxCheckResultControllerSpec
 
     "handling request to tax check result page " must {
 
-      def performAction(): Future[Result] = controller.taxCheckMatch(FakeRequest())
+      def performAction(): Future[Result]        = controller.taxCheckMatch(FakeRequest())
+      def performActionExpired(): Future[Result] = controller.taxCheckExpired(FakeRequest())
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/heclicensingbodyfrontend/controllers/TaxCheckResultControllerSpec.scala:71]: in the other controller specs we generally have 
```scala
"controller" when {
  "handling requests to x" must { ... }
  "handling requests to y" must { ... }
}
```
where `x` and `y` are the different endpoints in the controller. To be consistent with that it'd be neater to have a different area of the spec for each endpoint like the above. I generally like to collapse everything in the editor and navigate my way to the correct area with that convention in mind

File: test/uk/gov/hmrc/heclicensingbodyfrontend/services/HECTaxMatchServiceSpec.scala [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      "return an error" when {
+
+        "the http call fails" in {
+          mockMatchTaxCheck(taxCheckMatchRequest)(Left(Error("")))
+
+          val result = taxCheckService.matchTaxCheck(taxCheckMatchRequest)
+          await(result.value) shouldBe a[Left[_, _]]
+        }
+
+        "the http response does not come back with a non-ok (200) response" in {
```
Comment (meghana-viswanath) [inline test/uk/gov/hmrc/heclicensingbodyfrontend/services/HECTaxMatchServiceSpec.scala:77]: do you mean "the http response comes back with a non-ok (200) response" ?

**Review by meghana-viswanath — APPROVED** [review]

File: test/uk/gov/hmrc/heclicensingbodyfrontend/controllers/TaxCheckResultControllerSpec.scala (hunk 1 above)
Comment (priyankadhiman3006) [reply test/uk/gov/hmrc/heclicensingbodyfrontend/controllers/TaxCheckResultControllerSpec.scala:71]: sure, i can do that :)

File: test/uk/gov/hmrc/heclicensingbodyfrontend/services/HECTaxMatchServiceSpec.scala (hunk 2 above)
Comment (priyankadhiman3006) [reply test/uk/gov/hmrc/heclicensingbodyfrontend/services/HECTaxMatchServiceSpec.scala]: updated

File: test/uk/gov/hmrc/heclicensingbodyfrontend/controllers/TaxCheckResultControllerSpec.scala (hunk 1 above)
Comment (priyankadhiman3006) [reply test/uk/gov/hmrc/heclicensingbodyfrontend/controllers/TaxCheckResultControllerSpec.scala]: Done

**Review by AndyHWChung — APPROVED** [review]

**Pull request merged**